### PR TITLE
ansible: fix add support for juniper devices

### PR DESCRIPTION
**Pull request by gaige**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Add support for Juniper devices to the brew ansible distribution.  Specifically adds prerequisite packages that support the built-in junos_ modules.

### CODE REVIEW

File: Formula/ansible.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Ansible < Formula
   homepage "https://www.ansible.com/"
   url "https://releases.ansible.com/ansible/ansible-2.3.0.0.tar.gz"
   sha256 "299f3907cd566a20e163942fa82b6afc86ef89c2726ba503b90c1a651e82a458"
+  revision 2
```
Comment (ilovezfs) [inline Formula/ansible.rb]: `revision 1`

**Review by ilovezfs — CHANGES_REQUESTED** [review]
Also needs to be added to the comments at https://github.com/gaige/homebrew-core/blob/4640f6e07763621244a21a166f9252207f50daeb/Formula/ansible.rb#L23-L37

File: Formula/ansible.rb (hunk 1 above)
Comment (ilovezfs) [inline Formula/ansible.rb]: Now `revision 2` is actually correct because I shipped https://github.com/Homebrew/homebrew-core/pull/12791.

**Comment by ilovezfs** [comment]
This also needs conflicts resolved since https://github.com/Homebrew/homebrew-core/pull/12791 shipped.

File: Formula/ansible.rb (hunk 1 above)
Comment (gaige) [inline Formula/ansible.rb]: I was just about to ask that.  I'm re-running tests and will push in a couple of minutes.

**Comment by gaige** [comment]
@ilovezfs It wasn't clear from the contribution notes whether the community standard is to rebase or merge. I rebased to the latest before fixing the conflict.  If the standard here is for merges, I'll be happy to do that going forward.

File: Formula/ansible.rb (hunk 1 above)
Comment (gaige) [inline Formula/ansible.rb]: @ilovezfs I'm not sure how to clear this change request at this point. It's now out of date (as previously mentioned) but it's showing up as a red block on the pull request.

**Comment by ilovezfs** [comment]
Rebasing is the standard but `brew pull` will do that, so merge is usually acceptable as well, unless it's a `bottle :unneeded` PR with multiple commits.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
@gaige we usually squash to one commit, as you can see in https://github.com/Homebrew/homebrew-core/commit/7e70f15cb860c7e71da022f796dd961b4898039c.

Thanks for the PR!

**Comment by gaige** [comment]
Got it.  Will do so in the future.  Thanks for the speedy approval!

**Comment by ilovezfs** [comment]
@gaige you're welcome!

**Pull request closed without merging**